examples/community/README.md

```diff
@@ -75,6 +76,48 @@ pipe = DiffusionPipeline.from_pretrained("runwayml/stable-diffusion-v1-5", custo
 
 ## Example usages
 
+### HD-Painter
+
+Implementation of [HD-Painter: High-Resolution and Prompt-Faithful Text-Guided Image Inpainting with Diffusion Models](https://arxiv.org/abs/2312.14091).
+
+![teaser-img](https://lh3.googleusercontent.com/fife/AGXqzDnB8YYA3R_MLn31EzWV13TEUm1VOvP2RhbfLoIl4hulwN3W7GwzATZUSIQxiADAnt3DG07QD5BnDKnLR7bdJARv4FRJsXhqWT0AiWY8A_uoMI91dpPDA9RL18a42OSEQSAjYnGPy-9dY3JUZy-6nU6m721zbWfHM-EmNqhMdhaVlsx9997wIKMl8Rs3plL_PaZD8yWt7dqT61Y5h-kZz35KcedU0ZihnmXKBXbbqb1d9nzBHQi-d7EWR-MdTwDPDuN00vQW_S1KmXC_hWEBTJmpS-MST0m0UZzR1tTFRdQxiiZXkn3GnB5M1clOif_kXhC7ceaRfR7iwr1tyTGyyB9C03zmFiKiJS5ZunOXJ9wgKpMzprnae6Yosh-vfrZm4Y6ifGkpFt6-0kvJx0C7ohSpU4wKi3wZmbEYUnwNlTTlaY4aGIy7JsaCgzCzw0mKd7vKOSKnA4lUduJN3e9Qbnl-51ONgHe6Ed-MTiu0kJSTHvvjRQEiEo1klm93UE7puxhXdzBsv3SSdWgOjy07fCMEPgwHkT5osTtmgd9Cj_jYG7PvbfL5xiV2HggzgdTAdm589tLz2J--6ebFly64rC0w0zBXjglZieCXzbKonH3N1ne-FHFQDCQrLKBvPuvfVRMHUz1JqBb_gJZTUdArrG9k84yh4ASVx46U4HGn0aIr9QXbFge39xGCmpohje9544TZpqr74mi7pD4l8Oi4qHc68RpRvvSyd2QDaF451RItQxXJmTTtJKpLPuy9R-HxMhnh-Z9EcrWFA2V4JA_lK8KTAkS_occ-T-K0GFsHtwEijEgNURjAljhFM9ciyGluOPaJIN9_jMtCL6bhbktBe16EsewNNlaDyqUWL9pCnrI0Gcpnh0lg899fEZdG_flfWUQkHTpFkQlOrNLZux7FBvWrUm3yMgvaeQMH4GPlrENBpnUvjz3gWCkeKeDQ0CFrEaXkbqYk16YKdiPk3dBmdlOTul0hVswI9vxDWWKz2zxgids_x0TjJmLQmLlXDKJgznMHKRDtjEvEKDwxQqG_tH9H7Os7oBj3GcJidi3eae-XNrEG0qg8lI1ogqc1IHU56IJdNUoz_WulFQ0Crvn6sOITiy5_X66rsqT130-mRVvKOirkLuCX1Ae9pIkiUOwtY1KYFe6F4cwRuoGbMbw9-1HCjF0r46QqBKsxh4RkWRcn4EJtX-3yPxCWuuCNHIZlDTfo1OzQWO3LVp_9XNTTR5DW0eEyLzt1yQU9OWxyWbk3CYTG-ONVBVA-JzIknpmpzJskxz5_sQRi3_iZYmca7lCRqgGUpcG7ONkUeThgIaItHob-6oYZbG6yjECG_F2I4weGi2cbD2-ooVaY8HgqLTX5XBSalSqF9hWtxPZvtH-laIbHYp2SMDmGOsE_T7Zi8g9veg_1rDq84nWg0IOkPGQFA9pv26hI-Rdt1aFnKqXhUNW0yNza07X-p-sDgEKr3vBRwPFKvjqfYGJPnBN5HWJzKxBbMgfX90DSQHvVIyiJiZ0hA_pyT-1xpIGbSJ_wg2xAPko_LkJWhPuBfiQlCE9JnTVkmGG-hj_ugWbWdp9tWMMbcleKBy68qtecX5HPy61jmvTLTOU6fVIKBR4Pc8wawkaSohm3la2P54viMO0mcVmOj4ps7YDlbai-Rtr7gxrcAlqyIvbjDpqanRHHO-4Vm8CuOQx3IzcYsHglE8Eyt6CJBCR057KoaiS3kYprWvkILI8znlFHpWvakLO6cUQSyqVDvsXefpOL)
```

Comment:
Can you host this on a GitHub repo or maybe even in a repository under https://huggingface.co/PAIR organization and provide a link here?